pkg/generators/markers.go

```diff
@@ -240,15 +240,21 @@ func lintArrayIndices(markerComments []string) error {
 	prevIndex := -1
 	for _, line := range markerComments {
 		line = strings.Trim(line, " ")
-		subscriptIdx := strings.Index(line, "[")
+		split := strings.Split(line, "=")
+		if len(split) < 2 {
+			// Ignore malformed key in linter.
```

Comment:
no error?

Reply:
actually, just realized this is possible with bool, so in a sense it's not malformed?

The error would still get caught later on when validating empty CEL tags, not sure if you'd like to error earlier though?

Test case:

```
				`+k8s:validation:cel[0]:rule="oldSelf == self"`,
				`+k8s:validation:cel[6]:optionalOldSelf`,
```

Reply:
> The error would still get caught later on when validating empty CEL tags, not sure if you'd like to error earlier though?

The linter is not meant to catch those types of errors. Just avoid common mistakes labelling array indices

> actually, just realized this is possible with bool, so in a sense it's not malformed?

That's a good catch. Ill add a test and fix that case

Reply:
fixed & added tests